tools/extract-verification-info.sh

```diff
@@ -3,14 +3,42 @@
 img=$1
 
 if [ "$img" == "" ]; then
-  echo "Usage ./extract_verification_info.sh <verification.img>"
+  echo "Usage ./extract-verification-info.sh <USB_device_name|verification_img>"
+  echo "E.g., ./extract-verification-info.sh /dev/disk4"
+  echo "Or, ./extract-verification-info.sh dist/amd64/current/verification.raw"
   exit
 fi
 
-size=$(fdisk -l "$img" | grep raw5 | awk '{print $2}')
-mkdir /tmp/verification_mnt
-sudo mount -o offset=$((size*512)) "$img" /tmp/verification_mnt
-cp /tmp/verification_mnt/*/summary.log ./
-cat /tmp/verification_mnt/*/summary.log
-sudo umount /tmp/verification_mnt
-rmdir /tmp/verification_mnt
+checkOScmd="echo ${OSTYPE} | grep -q darwin" # Running on MacOS
+checkVerificationImg="eval file ${img} | grep -q DOS/MBR" # this is a file not a block device
```

Comment:
If you are trying to check if the file is a regular file or a block device, you don't need to use this whole command, you can just check it with the test command (it can be directly on the if):

-f : File exists and is a regular file
-b : File exists and is block device

```sh
if [ -b "${img}" ]; then
  echo "Block file"
elif [ -f "${img}" ]; then
  echo "Regular file"
fi
```